src/main/include/subsystems/Shooter.h

```diff
@@ -40,14 +42,18 @@ namespace t34
 
         void RunIntake(const double motor_output);
```

Comment:
As with the suggestion about `RunTopArmMotor()`, these might benefit from a similar convention:

```suggestion
        void RunShooter(const double percent);
        void RunIntake(const double percent);
```